This patch reverts the following commit, because OpenBSD base system only has
Unicode::Collate 1.27.

commit d5cdbe1f50e15288baabd95ea41c92fda32e190e
Author: Philip Kime <Philip@kime.org.uk>
Date:   Sat Nov 28 21:37:37 2020 +0100

    Update for U:C 1.29

diff --git Build.PL Build.PL
index 328d08be..73919e3e 100644
--- Build.PL
+++ Build.PL
@@ -58,7 +58,7 @@ my $builder = $class->new(
                  'Mozilla::CA' =>  '20160104',
                  'Regexp::Common' => 0,
                  'Log::Log4perl' => 0,
-                 'Unicode::Collate' => '1.29',
+                 'Unicode::Collate' => '1.27',
                  'Unicode::Normalize' => '1.26',
                  'Unicode::LineBreak' => '2019.001',
                  'Unicode::GCString' => 0,
diff --git t/annotations.t t/annotations.t
index 0d6643d0..c6f86c7f 100644
--- t/annotations.t
+++ t/annotations.t
@@ -73,7 +73,7 @@ my $ann1 = q|    \entry{ann1}{misc}{}
       \strng{authorfullhash}{90ae96c82de92e36949bc64254bbde0c}
       \field{extraname}{1}
       \field{sortinit}{L}
-      \field{sortinithash}{7c47d417cecb1f4bd38d1825c427a61a}
+      \field{sortinithash}{dad3efd0836470093a7b4a7bb756eb8c}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
       \field{title}{The Title}
@@ -116,7 +116,7 @@ my $ann2 = q|    \entry{ann2}{misc}{}
       \strng{authorfullhash}{90ae96c82de92e36949bc64254bbde0c}
       \field{extraname}{2}
       \field{sortinit}{L}
-      \field{sortinithash}{7c47d417cecb1f4bd38d1825c427a61a}
+      \field{sortinithash}{dad3efd0836470093a7b4a7bb756eb8c}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
       \field{title}{The Title}
diff --git t/basic-misc.t t/basic-misc.t
index 18e6ce84..9fa7bd7d 100644
--- t/basic-misc.t
+++ t/basic-misc.t
@@ -97,7 +97,7 @@ my $u1 = q|    \entry{u1}{misc}{}
       \strng{authorfullhash}{b78abdc838d79b6576f2ed0021642766}
       \field{labelalpha}{AAA\textbf{+}00}
       \field{sortinit}{A}
-      \field{sortinithash}{2f401846e2029bad6b3ecc16d50031e2}
+      \field{sortinithash}{a3dcedd53b04d1adfd5ac303ecd5e6fa}
       \true{singletitle}
       \true{uniquework}
       \field{labelnamesource}{author}
@@ -231,7 +231,7 @@ my $murray1 = q|    \entry{murray}{article}{}
       \strng{authorfullhash}{1572cc3fd324f560e5e71d041a6bd764}
       \field{labelalpha}{Hos\textbf{+}98}
       \field{sortinit}{H}
-      \field{sortinithash}{23a3aa7c24e56cfa16945d55545109b5}
+      \field{sortinithash}{6db6145dae8dc9e1271a8d556090b50a}
       \true{singletitle}
       \true{uniquework}
       \field{labelnamesource}{author}
@@ -348,7 +348,7 @@ my $murray2 = q|    \entry{murray}{article}{}
       \strng{authorfullhash}{1572cc3fd324f560e5e71d041a6bd764}
       \field{labelalpha}{Hos98}
       \field{sortinit}{H}
-      \field{sortinithash}{23a3aa7c24e56cfa16945d55545109b5}
+      \field{sortinithash}{6db6145dae8dc9e1271a8d556090b50a}
       \true{singletitle}
       \true{uniquework}
       \field{labelnamesource}{author}
@@ -389,7 +389,7 @@ my $t1 = q+    \entry{t1}{misc}{}
       \field{extraname}{1}
       \field{labelalpha}{Bro92}
       \field{sortinit}{B}
-      \field{sortinithash}{d7095fff47cda75ca2589920aae98399}
+      \field{sortinithash}{8de16967003c7207dae369d874f1456e}
       \true{uniquework}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -419,7 +419,7 @@ my $t2 = q|    \entry{t2}{misc}{}
       \field{extraname}{2}
       \field{labelalpha}{Bro94}
       \field{sortinit}{B}
-      \field{sortinithash}{d7095fff47cda75ca2589920aae98399}
+      \field{sortinithash}{8de16967003c7207dae369d874f1456e}
       \true{uniquework}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -452,7 +452,7 @@ my $anon1 = q|    \entry{anon1}{unpublished}{}
       \strng{shortauthorfullhash}{9873a6cc65c553faa2b21aaad626fe4b}
       \field{labelalpha}{XAn35}
       \field{sortinit}{A}
-      \field{sortinithash}{2f401846e2029bad6b3ecc16d50031e2}
+      \field{sortinithash}{a3dcedd53b04d1adfd5ac303ecd5e6fa}
       \true{singletitle}
       \true{uniquework}
       \field{labelnamesource}{shortauthor}
@@ -491,7 +491,7 @@ my $anon2 = q|    \entry{anon2}{unpublished}{}
       \strng{shortauthorfullhash}{f64c29e89ea49402b997956610b58ef6}
       \field{labelalpha}{YAn39}
       \field{sortinit}{A}
-      \field{sortinithash}{2f401846e2029bad6b3ecc16d50031e2}
+      \field{sortinithash}{a3dcedd53b04d1adfd5ac303ecd5e6fa}
       \true{singletitle}
       \true{uniquework}
       \field{labelnamesource}{shortauthor}
@@ -526,7 +526,7 @@ my $url1 = q|    \entry{url1}{misc}{}
       \field{extraname}{4}
       \field{labelalpha}{Ali05}
       \field{sortinit}{A}
-      \field{sortinithash}{2f401846e2029bad6b3ecc16d50031e2}
+      \field{sortinithash}{a3dcedd53b04d1adfd5ac303ecd5e6fa}
       \field{extraalpha}{4}
       \field{labelnamesource}{author}
       \field{year}{2005}
@@ -710,7 +710,7 @@ my $isbn1 = q|    \entry{isbn1}{misc}{}
       \field{extraname}{1}
       \field{labelalpha}{Flu}
       \field{sortinit}{F}
-      \field{sortinithash}{2638baaa20439f1b5a8f80c6c08a13b4}
+      \field{sortinithash}{fb0c0faa89eb6abae8213bf60e6799ea}
       \field{extraalpha}{1}
       \field{labelnamesource}{author}
       \field{isbn}{978-0-8165-2066-4}
@@ -735,7 +735,7 @@ my $isbn2 = q|    \entry{isbn2}{misc}{}
       \field{extraname}{2}
       \field{labelalpha}{Flu}
       \field{sortinit}{F}
-      \field{sortinithash}{2638baaa20439f1b5a8f80c6c08a13b4}
+      \field{sortinithash}{fb0c0faa89eb6abae8213bf60e6799ea}
       \field{extraalpha}{2}
       \field{labelnamesource}{author}
       \field{isbn}{978-0-8165-2066-4}
@@ -778,7 +778,7 @@ my $clone1 = q|    \entry{snk1}{book}{}
       \field{extraname}{2}
       \field{labelalpha}{vDoe}
       \field{sortinit}{v}
-      \field{sortinithash}{afb52128e5b4dc4b843768c0113d673b}
+      \field{sortinithash}{02432525618c08e2b03cac47c19764af}
       \field{extraalpha}{2}
       \field{labelnamesource}{author}
     \endentry
@@ -807,7 +807,7 @@ my $clone2 = q|    \entry{clone-snk1}{book}{}
       \field{extraname}{1}
       \field{labelalpha}{vDoe}
       \field{sortinit}{v}
-      \field{sortinithash}{afb52128e5b4dc4b843768c0113d673b}
+      \field{sortinithash}{02432525618c08e2b03cac47c19764af}
       \field{extraalpha}{1}
       \field{labelnamesource}{author}
       \field{addendum}{add}
@@ -839,7 +839,7 @@ my $ent1 = q|    \entry{ent1}{book}{}
       \strng{authorfullhash}{b2536a425d549b46de5f21c4d468050a}
       \field{labelalpha}{SdB}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \true{singletitle}
       \field{labelnamesource}{author}
     \endentry
@@ -862,7 +862,7 @@ my $verb1 = q|    \entry{verb1}{book}{}
       \strng{authorfullhash}{cac5a25f503e71f5ef28f474e14007b6}
       \field{labelalpha}{All}
       \field{sortinit}{A}
-      \field{sortinithash}{2f401846e2029bad6b3ecc16d50031e2}
+      \field{sortinithash}{a3dcedd53b04d1adfd5ac303ecd5e6fa}
       \true{singletitle}
       \field{labelnamesource}{author}
       \verb{verba}
diff --git t/biblatexml.t t/biblatexml.t
index 95b9644d..070cf937 100644
--- t/biblatexml.t
+++ t/biblatexml.t
@@ -111,7 +111,7 @@ my $l1 = q|    \entry{bltx1}{misc}{useprefix=false}
       \strng{translatornamehash}{b44eba830fe9817fbe8e53c82f1cbe04}
       \strng{translatorfullhash}{b44eba830fe9817fbe8e53c82f1cbe04}
       \field{sortinit}{v}
-      \field{sortinithash}{afb52128e5b4dc4b843768c0113d673b}
+      \field{sortinithash}{02432525618c08e2b03cac47c19764af}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -158,7 +158,7 @@ my $l1 = q|    \entry{bltx1}{misc}{useprefix=false}
 
 my $l2 = q|    \entry{loopkey:a}{book}{}
       \field{sortinit}{0}
-      \field{sortinithash}{c5602f03f17cc894ea7a6362c3cb0e13}
+      \field{sortinithash}{bcf7a1f4afb88b7299f988caccb80d1c}
     \endentry
 |;
 
diff --git t/crossrefs.t t/crossrefs.t
index 76866574..bc39abf2 100644
--- t/crossrefs.t
+++ t/crossrefs.t
@@ -83,7 +83,7 @@ my $cr1 = q|    \entry{cr1}{inbook}{}
       \strng{editornamehash}{c129df5593fdaa7475548811bfbb227d}
       \strng{editorfullhash}{c129df5593fdaa7475548811bfbb227d}
       \field{sortinit}{G}
-      \field{sortinithash}{32d67eca0634bf53703493fb1090a2e8}
+      \field{sortinithash}{62eb2aa29549e4fdbd3cb154ec5711cb}
       \true{singletitle}
       \true{uniquetitle}
       \true{uniquework}
@@ -133,7 +133,7 @@ my $cr2 = q|    \entry{cr2}{inbook}{}
       \strng{editornamehash}{c129df5593fdaa7475548811bfbb227d}
       \strng{editorfullhash}{c129df5593fdaa7475548811bfbb227d}
       \field{sortinit}{F}
-      \field{sortinithash}{2638baaa20439f1b5a8f80c6c08a13b4}
+      \field{sortinithash}{fb0c0faa89eb6abae8213bf60e6799ea}
       \true{singletitle}
       \true{uniquetitle}
       \true{uniquework}
@@ -167,7 +167,7 @@ my $cr_m = q|    \entry{cr_m}{book}{}
       \strng{editornamehash}{c129df5593fdaa7475548811bfbb227d}
       \strng{editorfullhash}{c129df5593fdaa7475548811bfbb227d}
       \field{sortinit}{G}
-      \field{sortinithash}{32d67eca0634bf53703493fb1090a2e8}
+      \field{sortinithash}{62eb2aa29549e4fdbd3cb154ec5711cb}
       \true{crossrefsource}
       \true{uniquetitle}
       \field{labeltitlesource}{title}
@@ -205,7 +205,7 @@ my $cr3 = q|    \entry{cr3}{inbook}{}
       \strng{editornamehash}{a1f5c22413396d599ec766725b226735}
       \strng{editorfullhash}{a1f5c22413396d599ec766725b226735}
       \field{sortinit}{A}
-      \field{sortinithash}{2f401846e2029bad6b3ecc16d50031e2}
+      \field{sortinithash}{a3dcedd53b04d1adfd5ac303ecd5e6fa}
       \true{singletitle}
       \true{uniquetitle}
       \true{uniquework}
@@ -250,7 +250,7 @@ my $cr4 = q|    \entry{cr4}{inbook}{}
       \strng{editornamehash}{6ea89bd4958743a20b70fe17647d6af5}
       \strng{editorfullhash}{6ea89bd4958743a20b70fe17647d6af5}
       \field{sortinit}{M}
-      \field{sortinithash}{4625c616857f13d17ce56f7d4f97d451}
+      \field{sortinithash}{2e5c2f51f7fa2d957f3206819bf86dc3}
       \true{singletitle}
       \true{uniquetitle}
       \true{uniquework}
@@ -281,7 +281,7 @@ my $crt = q|    \entry{crt}{book}{}
       \strng{editornamehash}{a1f5c22413396d599ec766725b226735}
       \strng{editorfullhash}{a1f5c22413396d599ec766725b226735}
       \field{sortinit}{B}
-      \field{sortinithash}{d7095fff47cda75ca2589920aae98399}
+      \field{sortinithash}{8de16967003c7207dae369d874f1456e}
       \true{uniquetitle}
       \field{labeltitlesource}{title}
       \field{title}{Beasts of the Burbling Burns}
@@ -317,7 +317,7 @@ my $cr6 = q|    \entry{cr6}{inproceedings}{}
       \strng{editorfullhash}{344a7f427fb765610ef96eb7bce95257}
       \field{extraname}{2}
       \field{sortinit}{A}
-      \field{sortinithash}{2f401846e2029bad6b3ecc16d50031e2}
+      \field{sortinithash}{a3dcedd53b04d1adfd5ac303ecd5e6fa}
       \true{uniquetitle}
       \true{uniquework}
       \field{labelnamesource}{author}
@@ -370,7 +370,7 @@ my $cr7 = q|    \entry{cr7}{inbook}{}
       \strng{bookauthorfullhash}{91a1dd4aeed3c4ec29ca74c4e778be5f}
       \field{extraname}{1}
       \field{sortinit}{A}
-      \field{sortinithash}{2f401846e2029bad6b3ecc16d50031e2}
+      \field{sortinithash}{a3dcedd53b04d1adfd5ac303ecd5e6fa}
       \true{uniquetitle}
       \true{uniquework}
       \field{labelnamesource}{author}
@@ -405,7 +405,7 @@ my $cr8 = q|    \entry{cr8}{incollection}{}
       \strng{authorfullhash}{3d449e56eb3ca1ae80dc99a18d689795}
       \field{extraname}{4}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \true{singletitle}
       \true{uniquetitle}
       \true{uniquework}
@@ -435,7 +435,7 @@ my $xr1 = q|    \entry{xr1}{inbook}{}
       \strng{authornamehash}{e0ecc4fc668ee499d1afba44e1ac064d}
       \strng{authorfullhash}{e0ecc4fc668ee499d1afba44e1ac064d}
       \field{sortinit}{Z}
-      \field{sortinithash}{96892c0b0a36bb8557c40c49813d48b3}
+      \field{sortinithash}{8f7b480688e809b50b6f6577b16f3db5}
       \true{singletitle}
       \true{uniquetitle}
       \true{uniquework}
@@ -464,7 +464,7 @@ my $xr2 = q|    \entry{xr2}{inbook}{}
       \strng{authornamehash}{6afa09374ecfd6b394ce714d2d9709c7}
       \strng{authorfullhash}{6afa09374ecfd6b394ce714d2d9709c7}
       \field{sortinit}{I}
-      \field{sortinithash}{8d291c51ee89b6cd86bf5379f0b151d8}
+      \field{sortinithash}{9417e9a1288a9371e2691d999083ed39}
       \true{singletitle}
       \true{uniquetitle}
       \true{uniquework}
@@ -495,7 +495,7 @@ my $xrm = q|    \entry{xrm}{book}{}
       \strng{editornamehash}{809950f9b59ae207092b909a19dcb27b}
       \strng{editorfullhash}{809950f9b59ae207092b909a19dcb27b}
       \field{sortinit}{C}
-      \field{sortinithash}{4d103a86280481745c9c897c925753c0}
+      \field{sortinithash}{4c244ceae61406cdc0cc2ce1cb1ff703}
       \true{xrefsource}
       \true{uniquetitle}
       \field{labeltitlesource}{title}
@@ -520,7 +520,7 @@ my $xr3 = q|    \entry{xr3}{inbook}{}
       \strng{authornamehash}{9788055665b9bb4b37c776c3f6b74f16}
       \strng{authorfullhash}{9788055665b9bb4b37c776c3f6b74f16}
       \field{sortinit}{N}
-      \field{sortinithash}{22369a73d5f88983a108b63f07f37084}
+      \field{sortinithash}{98cf339a479c0454fe09153a08675a15}
       \true{singletitle}
       \true{uniquetitle}
       \true{uniquework}
@@ -550,7 +550,7 @@ my $xrt = q|    \entry{xrt}{book}{}
       \strng{editornamehash}{bf7d6b02f3e073913e5bfe5059508dd5}
       \strng{editorfullhash}{bf7d6b02f3e073913e5bfe5059508dd5}
       \field{sortinit}{K}
-      \field{sortinithash}{c02bf6bff1c488450c352b40f5d853ab}
+      \field{sortinithash}{d3edc18d54b9438a72c24c925bfb38f4}
       \true{uniquetitle}
       \field{labeltitlesource}{title}
       \field{title}{Kings, Cork and Calculation}
@@ -576,7 +576,7 @@ my $xr4 = q|    \entry{xr4}{inbook}{}
       \strng{authorfullhash}{7804ffef086c0c4686c235807f5cb502}
       \field{extraname}{1}
       \field{sortinit}{M}
-      \field{sortinithash}{4625c616857f13d17ce56f7d4f97d451}
+      \field{sortinithash}{2e5c2f51f7fa2d957f3206819bf86dc3}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
       \field{origyear}{1933}
@@ -604,7 +604,7 @@ my $mxr = q|    \entry{mxr}{inbook}{}
       \strng{authorfullhash}{7804ffef086c0c4686c235807f5cb502}
       \field{extraname}{2}
       \field{sortinit}{M}
-      \field{sortinithash}{4625c616857f13d17ce56f7d4f97d451}
+      \field{sortinithash}{2e5c2f51f7fa2d957f3206819bf86dc3}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
       \field{origyear}{1933}
@@ -629,7 +629,7 @@ my $mcr = q|    \entry{mcr}{inbook}{}
       \strng{authorfullhash}{7804ffef086c0c4686c235807f5cb502}
       \field{extraname}{3}
       \field{sortinit}{M}
-      \field{sortinithash}{4625c616857f13d17ce56f7d4f97d451}
+      \field{sortinithash}{2e5c2f51f7fa2d957f3206819bf86dc3}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
       \field{origyear}{1933}
@@ -664,7 +664,7 @@ my $ccr1 = q|    \entry{ccr2}{book}{}
       \strng{editorfullhash}{cfee758a1c82df2e26af1985e061bb0a}
       \field{extraname}{1}
       \field{sortinit}{V}
-      \field{sortinithash}{afb52128e5b4dc4b843768c0113d673b}
+      \field{sortinithash}{02432525618c08e2b03cac47c19764af}
       \true{uniquetitle}
       \true{uniquework}
       \field{labelnamesource}{author}
@@ -698,7 +698,7 @@ my $ccr2 = q|    \entry{ccr3}{inbook}{}
       \strng{editornamehash}{cfee758a1c82df2e26af1985e061bb0a}
       \strng{editorfullhash}{cfee758a1c82df2e26af1985e061bb0a}
       \field{sortinit}{P}
-      \field{sortinithash}{ff3bcf24f47321b42cb156c2cc8a8422}
+      \field{sortinithash}{bb5b15f2db90f7aef79bb9e83defefcb}
       \true{uniquetitle}
       \field{labeltitlesource}{title}
       \field{booktitle}{Misc etc.}
@@ -730,7 +730,7 @@ my $ccr3 = q|    \entry{ccr4}{inbook}{}
 
 my $s1 = q|    \entry{s1}{inbook}{}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \true{uniquetitle}
       \field{labeltitlesource}{title}
       \strng{crossref}{s2}
@@ -764,7 +764,7 @@ my $xc2 = q|    \entry{xc2}{inbook}{}
       \strng{bookauthorfullhash}{1a0f7d518cccdad859a74412ef956474}
       \field{extraname}{2}
       \field{sortinit}{C}
-      \field{sortinithash}{4d103a86280481745c9c897c925753c0}
+      \field{sortinithash}{4c244ceae61406cdc0cc2ce1cb1ff703}
       \true{xrefsource}
       \field{labelnamesource}{author}
       \field{booktitle}{Title}
@@ -773,7 +773,7 @@ my $xc2 = q|    \entry{xc2}{inbook}{}
 
 my $b1 = q|    \entry{b1}{inbook}{}
       \field{sortinit}{2}
-      \field{sortinithash}{8b555b3791beccb63322c22f3320aa9a}
+      \field{sortinithash}{ed39bb39cf854d5250e95b1c1f94f4ed}
       \strng{crossref}{b2}
       \field{day}{3}
       \field{month}{3}
@@ -807,7 +807,7 @@ my $sup1 = q|    \entry{sup1}{mvbook}{}
       \strng{authorfullhash}{556c8dba145b472e6a8598d506f7cbe2}
       \field{extraname}{3}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \true{crossrefsource}
       \true{singletitle}
       \field{labelnamesource}{author}
@@ -832,7 +832,7 @@ my $sup2 = q|    \entry{sup2}{book}{}
       \strng{authorfullhash}{556c8dba145b472e6a8598d506f7cbe2}
       \field{extraname}{1}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \true{singletitle}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
diff --git t/datalists.t t/datalists.t
index 812b768f..a897c9f6 100644
--- t/datalists.t
+++ t/datalists.t
@@ -187,7 +187,7 @@ my $K11 = q|    \entry{K11}{book}{}
       \strng{authornamehash}{4edc280a0ef229f9c061e3b121b17482}
       \strng{authorfullhash}{4edc280a0ef229f9c061e3b121b17482}
       \field{sortinit}{a}
-      \field{sortinithash}{2f401846e2029bad6b3ecc16d50031e2}
+      \field{sortinithash}{a3dcedd53b04d1adfd5ac303ecd5e6fa}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -219,7 +219,7 @@ my $K12 = q|    \entry{K12}{book}{}
       \strng{authornamehash}{a846a485fc9cbb59b0ebeedd6ac637e4}
       \strng{authorfullhash}{a846a485fc9cbb59b0ebeedd6ac637e4}
       \field{sortinit}{Z}
-      \field{sortinithash}{96892c0b0a36bb8557c40c49813d48b3}
+      \field{sortinithash}{8f7b480688e809b50b6f6577b16f3db5}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
diff --git t/dateformats.t t/dateformats.t
index 9c179f06..1eca838b 100644
--- t/dateformats.t
+++ t/dateformats.t
@@ -89,7 +89,7 @@ my $l13c = q|    \entry{L13}{book}{}
       \strng{authorfullhash}{8c77336299b25bdada7bf8038f46722f}
       \field{extraname}{3}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -127,7 +127,7 @@ my $l14 = q|    \entry{L14}{book}{}
       \strng{authorfullhash}{8c77336299b25bdada7bf8038f46722f}
       \field{extraname}{4}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extradate}{3}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
@@ -169,7 +169,7 @@ my $l15 = q|    \entry{L15}{book}{}
       \strng{authorfullhash}{8c77336299b25bdada7bf8038f46722f}
       \field{extraname}{12}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extradate}{4}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -202,7 +202,7 @@ my $l16 = q|    \entry{L16}{proceedings}{}
       \strng{editorfullhash}{8c77336299b25bdada7bf8038f46722f}
       \field{extraname}{13}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extradate}{7}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{event}
@@ -241,7 +241,7 @@ my $l17 = q|    \entry{L17}{proceedings}{}
       \strng{editorfullhash}{8c77336299b25bdada7bf8038f46722f}
       \field{extraname}{5}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extradate}{4}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
@@ -300,7 +300,7 @@ my $l17c = q|    \entry{L17}{proceedings}{}
       \strng{editorfullhash}{8c77336299b25bdada7bf8038f46722f}
       \field{extraname}{5}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{orig}
       \field{labelnamesource}{editor}
@@ -358,7 +358,7 @@ my $l17e = q|    \entry{L17}{proceedings}{}
       \strng{editorfullhash}{8c77336299b25bdada7bf8038f46722f}
       \field{extraname}{5}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{event}
       \field{labelnamesource}{editor}
@@ -480,7 +480,7 @@ my $era1 = q|    \entry{era1}{article}{}
       \strng{authorfullhash}{556c8dba145b472e6a8598d506f7cbe2}
       \field{extraname}{9}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -513,7 +513,7 @@ my $era2 = q|    \entry{era2}{inproceedings}{}
       \strng{authorfullhash}{556c8dba145b472e6a8598d506f7cbe2}
       \field{extraname}{10}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -547,7 +547,7 @@ my $era3 = q|    \entry{era3}{inproceedings}{}
       \strng{authorfullhash}{556c8dba145b472e6a8598d506f7cbe2}
       \field{extraname}{11}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -581,7 +581,7 @@ my $era4 = q|    \entry{era4}{inproceedings}{}
       \strng{authorfullhash}{556c8dba145b472e6a8598d506f7cbe2}
       \field{extraname}{6}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -623,7 +623,7 @@ my $time1 = q|    \entry{time1}{article}{}
       \strng{authorfullhash}{556c8dba145b472e6a8598d506f7cbe2}
       \field{extraname}{2}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -672,7 +672,7 @@ my $range1 = q|    \entry{range1}{inproceedings}{}
       \strng{authorfullhash}{556c8dba145b472e6a8598d506f7cbe2}
       \field{extraname}{7}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradate}{1}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
@@ -713,7 +713,7 @@ my $range2 = q|    \entry{range2}{inproceedings}{}
       \strng{authorfullhash}{556c8dba145b472e6a8598d506f7cbe2}
       \field{extraname}{8}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradate}{2}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
@@ -754,7 +754,7 @@ my $season1 = q|    \entry{season1}{inproceedings}{}
       \strng{authorfullhash}{556c8dba145b472e6a8598d506f7cbe2}
       \field{extraname}{1}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -786,7 +786,7 @@ my $unspec1 = q|    \entry{unspec1}{inproceedings}{}
       \strng{authorfullhash}{556c8dba145b472e6a8598d506f7cbe2}
       \field{extraname}{4}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -839,7 +839,7 @@ my $unspec2 = q|    \entry{unspec2}{article}{}
       \strng{authorfullhash}{556c8dba145b472e6a8598d506f7cbe2}
       \field{extraname}{3}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
diff --git t/encoding.t t/encoding.t
index 1ec98064..12254a5c 100644
--- t/encoding.t
+++ t/encoding.t
@@ -73,7 +73,7 @@ my $encode1 = q|% $ biblatex auxiliary file $
       \strng{authorfullhash}{06a47edae2e847800cfd78323a0e6be8}
       \field{labelalpha}{Enc99}
       \field{sortinit}{E}
-      \field{sortinithash}{8da8a182d344d5b9047633dfc0cc9131}
+      \field{sortinithash}{c554bd1a0b76ea92b9f105fe36d9c7b0}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{year}
       \true{singletitle}
@@ -129,7 +129,7 @@ my $encode2 = q|% $ biblatex auxiliary file $
       \strng{authorfullhash}{06a47edae2e847800cfd78323a0e6be8}
       \field{labelalpha}{Enc99}
       \field{sortinit}{E}
-      \field{sortinithash}{8da8a182d344d5b9047633dfc0cc9131}
+      \field{sortinithash}{c554bd1a0b76ea92b9f105fe36d9c7b0}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{year}
       \true{singletitle}
@@ -185,7 +185,7 @@ my $encode3 = q|% $ biblatex auxiliary file $
       \strng{authorfullhash}{06a47edae2e847800cfd78323a0e6be8}
       \field{labelalpha}{Enc99}
       \field{sortinit}{E}
-      \field{sortinithash}{8da8a182d344d5b9047633dfc0cc9131}
+      \field{sortinithash}{c554bd1a0b76ea92b9f105fe36d9c7b0}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{year}
       \true{singletitle}
@@ -241,7 +241,7 @@ my $encode5 = q|% $ biblatex auxiliary file $
       \strng{authorfullhash}{06a47edae2e847800cfd78323a0e6be8}
       \field{labelalpha}{Enc99}
       \field{sortinit}{E}
-      \field{sortinithash}{8da8a182d344d5b9047633dfc0cc9131}
+      \field{sortinithash}{c554bd1a0b76ea92b9f105fe36d9c7b0}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{year}
       \true{singletitle}
@@ -297,7 +297,7 @@ my $encode6 = q|% $ biblatex auxiliary file $
       \strng{authorfullhash}{06a47edae2e847800cfd78323a0e6be8}
       \field{labelalpha}{Enc99}
       \field{sortinit}{E}
-      \field{sortinithash}{8da8a182d344d5b9047633dfc0cc9131}
+      \field{sortinithash}{c554bd1a0b76ea92b9f105fe36d9c7b0}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{year}
       \true{singletitle}
@@ -353,7 +353,7 @@ my $encode7 = q|% $ biblatex auxiliary file $
       \strng{authorfullhash}{06a47edae2e847800cfd78323a0e6be8}
       \field{labelalpha}{Enc99}
       \field{sortinit}{E}
-      \field{sortinithash}{8da8a182d344d5b9047633dfc0cc9131}
+      \field{sortinithash}{c554bd1a0b76ea92b9f105fe36d9c7b0}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{year}
       \true{singletitle}
diff --git t/names.t t/names.t
index f277a821..3120b84b 100644
--- t/names.t
+++ t/names.t
@@ -194,7 +194,7 @@ my $l1 = q|    \entry{L1}{book}{}
       \strng{authornamehash}{72287a68c1714cb1b9f4ab9e03a88b96}
       \strng{authorfullhash}{72287a68c1714cb1b9f4ab9e03a88b96}
       \field{sortinit}{A}
-      \field{sortinithash}{2f401846e2029bad6b3ecc16d50031e2}
+      \field{sortinithash}{a3dcedd53b04d1adfd5ac303ecd5e6fa}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -214,7 +214,7 @@ my $l2 = q|    \entry{L2}{book}{}
       \strng{authornamehash}{2098d59d0f19a2e003ee06c1aa750d57}
       \strng{authorfullhash}{2098d59d0f19a2e003ee06c1aa750d57}
       \field{sortinit}{B}
-      \field{sortinithash}{d7095fff47cda75ca2589920aae98399}
+      \field{sortinithash}{8de16967003c7207dae369d874f1456e}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -234,7 +234,7 @@ my $l3 = q|    \entry{L3}{book}{}
       \strng{authornamehash}{c8b06fe88bde128b25eb0b3b1cc5837c}
       \strng{authorfullhash}{c8b06fe88bde128b25eb0b3b1cc5837c}
       \field{sortinit}{C}
-      \field{sortinithash}{4d103a86280481745c9c897c925753c0}
+      \field{sortinithash}{4c244ceae61406cdc0cc2ce1cb1ff703}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -254,7 +254,7 @@ my $l4 = q|    \entry{L4}{book}{}
       \strng{authornamehash}{5ec958b850c0c2de7de7c42c84b9c419}
       \strng{authorfullhash}{5ec958b850c0c2de7de7c42c84b9c419}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -276,7 +276,7 @@ my $l5 = q|    \entry{L5}{book}{}
       \strng{authornamehash}{c6b9d281cc1ff3f35570f76f463d4244}
       \strng{authorfullhash}{c6b9d281cc1ff3f35570f76f463d4244}
       \field{sortinit}{v}
-      \field{sortinithash}{afb52128e5b4dc4b843768c0113d673b}
+      \field{sortinithash}{02432525618c08e2b03cac47c19764af}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -298,7 +298,7 @@ my $l6 = q|    \entry{L6}{book}{}
       \strng{authornamehash}{5fd24d3d1608a310ec205a6b201a5495}
       \strng{authorfullhash}{5fd24d3d1608a310ec205a6b201a5495}
       \field{sortinit}{v}
-      \field{sortinithash}{afb52128e5b4dc4b843768c0113d673b}
+      \field{sortinithash}{02432525618c08e2b03cac47c19764af}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -320,7 +320,7 @@ my $l7 = q|    \entry{L7}{book}{}
       \strng{authornamehash}{98edb0b90251df22b74328d9227eceb7}
       \strng{authorfullhash}{98edb0b90251df22b74328d9227eceb7}
       \field{sortinit}{v}
-      \field{sortinithash}{afb52128e5b4dc4b843768c0113d673b}
+      \field{sortinithash}{02432525618c08e2b03cac47c19764af}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -342,7 +342,7 @@ my $l8 = q|    \entry{L8}{book}{}
       \strng{authornamehash}{1211dc8dbbc191cbcab4da3c3c1fc48a}
       \strng{authorfullhash}{1211dc8dbbc191cbcab4da3c3c1fc48a}
       \field{sortinit}{v}
-      \field{sortinithash}{afb52128e5b4dc4b843768c0113d673b}
+      \field{sortinithash}{02432525618c08e2b03cac47c19764af}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -362,7 +362,7 @@ my $l9 = q|    \entry{L9}{book}{}
       \strng{authornamehash}{bae61a889ab149a6deafe45333204cf0}
       \strng{authorfullhash}{bae61a889ab149a6deafe45333204cf0}
       \field{sortinit}{I}
-      \field{sortinithash}{8d291c51ee89b6cd86bf5379f0b151d8}
+      \field{sortinithash}{9417e9a1288a9371e2691d999083ed39}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -385,7 +385,7 @@ my $l10 = q|    \entry{L10}{book}{}
       \strng{authornamehash}{37b4325752e394ddfb2fc810f6c88e27}
       \strng{authorfullhash}{37b4325752e394ddfb2fc810f6c88e27}
       \field{sortinit}{J}
-      \field{sortinithash}{b2f54a9081ace9966a7cb9413811edb4}
+      \field{sortinithash}{c45040a764d616897e7f5b30174d7b92}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -408,7 +408,7 @@ my $l10a = q|    \entry{L10a}{book}{}
       \strng{authornamehash}{7bf2c9d8b89a1930ee91bfddcaf20c9c}
       \strng{authorfullhash}{7bf2c9d8b89a1930ee91bfddcaf20c9c}
       \field{sortinit}{P}
-      \field{sortinithash}{ff3bcf24f47321b42cb156c2cc8a8422}
+      \field{sortinithash}{bb5b15f2db90f7aef79bb9e83defefcb}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -433,7 +433,7 @@ my $l11 = q|    \entry{L11}{book}{}
       \strng{authornamehash}{9f48d231be68c9435fab4faca55a5caf}
       \strng{authorfullhash}{9f48d231be68c9435fab4faca55a5caf}
       \field{sortinit}{v}
-      \field{sortinithash}{afb52128e5b4dc4b843768c0113d673b}
+      \field{sortinithash}{02432525618c08e2b03cac47c19764af}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -455,7 +455,7 @@ my $l12 = q|    \entry{L12}{book}{}
       \strng{authornamehash}{d7ca88c13a8f7ce1c23e920010a31f83}
       \strng{authorfullhash}{d7ca88c13a8f7ce1c23e920010a31f83}
       \field{sortinit}{d}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \true{uniqueprimaryauthor}
       \field{labelnamesource}{author}
     \endentry
@@ -476,7 +476,7 @@ my $l13 = q|    \entry{L13}{book}{}
       \strng{authornamehash}{227ac48bb788a658cfaa4eefc71ff0cc}
       \strng{authorfullhash}{227ac48bb788a658cfaa4eefc71ff0cc}
       \field{sortinit}{V}
-      \field{sortinithash}{afb52128e5b4dc4b843768c0113d673b}
+      \field{sortinithash}{02432525618c08e2b03cac47c19764af}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -496,7 +496,7 @@ my $l14 = q|    \entry{L14}{book}{}
       \strng{authornamehash}{779475052c17ed56dc3be900d0dfdf87}
       \strng{authorfullhash}{779475052c17ed56dc3be900d0dfdf87}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -519,7 +519,7 @@ my $l15 = q|    \entry{L15}{book}{}
       \strng{authorfullhash}{783c636e853e47a854ae034ebe9dde62}
       \field{extraname}{1}
       \field{sortinit}{v}
-      \field{sortinithash}{afb52128e5b4dc4b843768c0113d673b}
+      \field{sortinithash}{02432525618c08e2b03cac47c19764af}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -542,7 +542,7 @@ my $l16 = q|    \entry{L16}{book}{}
       \strng{authorfullhash}{783c636e853e47a854ae034ebe9dde62}
       \field{extraname}{2}
       \field{sortinit}{v}
-      \field{sortinithash}{afb52128e5b4dc4b843768c0113d673b}
+      \field{sortinithash}{02432525618c08e2b03cac47c19764af}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -563,7 +563,7 @@ my $l17 = q|    \entry{L17}{book}{}
       \strng{authorfullhash}{b51f667a3384d92ea5458ba80716bff7}
       \field{extraname}{1}
       \field{sortinit}{L}
-      \field{sortinithash}{7c47d417cecb1f4bd38d1825c427a61a}
+      \field{sortinithash}{dad3efd0836470093a7b4a7bb756eb8c}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -584,7 +584,7 @@ my $l18 = q|    \entry{L18}{book}{}
       \strng{authorfullhash}{b51f667a3384d92ea5458ba80716bff7}
       \field{extraname}{2}
       \field{sortinit}{L}
-      \field{sortinithash}{7c47d417cecb1f4bd38d1825c427a61a}
+      \field{sortinithash}{dad3efd0836470093a7b4a7bb756eb8c}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -604,7 +604,7 @@ my $l19 = q|    \entry{L19}{book}{}
       \strng{authornamehash}{83caa52f21f97e572dd3267bdf62978a}
       \strng{authorfullhash}{83caa52f21f97e572dd3267bdf62978a}
       \field{sortinit}{M}
-      \field{sortinithash}{4625c616857f13d17ce56f7d4f97d451}
+      \field{sortinithash}{2e5c2f51f7fa2d957f3206819bf86dc3}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -624,7 +624,7 @@ my $l19a = q|    \entry{L19a}{book}{}
       \strng{authornamehash}{0963f6904ccfeaac2770c5882a587001}
       \strng{authorfullhash}{0963f6904ccfeaac2770c5882a587001}
       \field{sortinit}{L}
-      \field{sortinithash}{7c47d417cecb1f4bd38d1825c427a61a}
+      \field{sortinithash}{dad3efd0836470093a7b4a7bb756eb8c}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -645,7 +645,7 @@ my $l20 = q|    \entry{L20}{book}{}
       \strng{authornamehash}{5f26c2f3b33095d5b005714893f4d698}
       \strng{authorfullhash}{5f26c2f3b33095d5b005714893f4d698}
       \field{sortinit}{F}
-      \field{sortinithash}{2638baaa20439f1b5a8f80c6c08a13b4}
+      \field{sortinithash}{fb0c0faa89eb6abae8213bf60e6799ea}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -666,7 +666,7 @@ my $l21 = q|    \entry{L21}{book}{}
       \strng{authorfullhash}{4389a3c0dc7da74487b50808ba9436ad}
       \field{extraname}{1}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \true{uniqueprimaryauthor}
       \field{labelnamesource}{author}
     \endentry
@@ -688,7 +688,7 @@ my $l22u = q|    \entry{L22}{book}{}
       \strng{authorfullhash}{e58b861545799d0eaf883402a882126e}
       \field{extraname}{1}
       \field{sortinit}{Š}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -710,7 +710,7 @@ my $l22 = q|    \entry{L22}{book}{}
       \strng{authorfullhash}{e58b861545799d0eaf883402a882126e}
       \field{extraname}{1}
       \field{sortinit}{\v{S}}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \true{uniqueprimaryauthor}
       \field{labelnamesource}{author}
     \endentry
@@ -733,7 +733,7 @@ my $l23 = q|    \entry{L23}{book}{}
       \strng{authorfullhash}{4389a3c0dc7da74487b50808ba9436ad}
       \field{extraname}{2}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -754,7 +754,7 @@ my $l24 = q|    \entry{L24}{book}{}
       \strng{authorfullhash}{e58b861545799d0eaf883402a882126e}
       \field{extraname}{2}
       \field{sortinit}{Š}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -772,7 +772,7 @@ my $l25 = q|    \entry{L25}{book}{}
       \strng{authornamehash}{d7cd2c5ea0848abc3e90609558b84a45}
       \strng{authorfullhash}{d7cd2c5ea0848abc3e90609558b84a45}
       \field{sortinit}{A}
-      \field{sortinithash}{2f401846e2029bad6b3ecc16d50031e2}
+      \field{sortinithash}{a3dcedd53b04d1adfd5ac303ecd5e6fa}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -790,7 +790,7 @@ my $l26 = q|    \entry{L26}{book}{}
       \strng{authornamehash}{8eee1dbafdbd0a4b73157e60f18b4784}
       \strng{authorfullhash}{8eee1dbafdbd0a4b73157e60f18b4784}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -808,7 +808,7 @@ my $l29 = q|    \entry{L29}{book}{}
       \strng{authornamehash}{27ad192a3a715aa89152b2a4ee392e8c}
       \strng{authorfullhash}{27ad192a3a715aa89152b2a4ee392e8c}
       \field{sortinit}{U}
-      \field{sortinithash}{6901a00e45705986ee5e7ca9fd39adca}
+      \field{sortinithash}{77a6935510e008adcf5b555e7b4f0711}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -848,7 +848,7 @@ my $l31 = q|    \entry{L31}{book}{}
       \strng{translatornamehash}{29c3ff92fff79d09a8b44d2f775de0b1}
       \strng{translatorfullhash}{29c3ff92fff79d09a8b44d2f775de0b1}
       \field{sortinit}{\~{Z}}
-      \field{sortinithash}{96892c0b0a36bb8557c40c49813d48b3}
+      \field{sortinithash}{8f7b480688e809b50b6f6577b16f3db5}
       \true{uniqueprimaryauthor}
       \field{labelnamesource}{author}
     \endentry
diff --git t/options.t t/options.t
index 1ccba9f0..44071345 100644
--- t/options.t
+++ t/options.t
@@ -100,7 +100,7 @@ my $l1 = q|    \entry{L1}{book}{}
       \strng{authornamehash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -137,7 +137,7 @@ my $l2 = q|    \entry{L2}{book}{maxalphanames=10,maxbibnames=3,maxcitenames=3,ma
       \strng{authornamehash}{19eec87c959944d6d9c72434a42856ba}
       \strng{authorfullhash}{19eec87c959944d6d9c72434a42856ba}
       \field{sortinit}{E}
-      \field{sortinithash}{8da8a182d344d5b9047633dfc0cc9131}
+      \field{sortinithash}{c554bd1a0b76ea92b9f105fe36d9c7b0}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -169,7 +169,7 @@ my $l3 = q|    \entry{L3}{book}{blah=10}
       \strng{authornamehash}{490250da1f3b92580d97563dc96c6c84}
       \strng{authorfullhash}{490250da1f3b92580d97563dc96c6c84}
       \field{sortinit}{B}
-      \field{sortinithash}{d7095fff47cda75ca2589920aae98399}
+      \field{sortinithash}{8de16967003c7207dae369d874f1456e}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
diff --git t/related-entries.t t/related-entries.t
index a2debdee..ad018d08 100644
--- t/related-entries.t
+++ t/related-entries.t
@@ -56,7 +56,7 @@ my $k1 = q|    \entry{key1}{article}{}
       \strng{authorfullhash}{a517747c3d12f99244ae598910d979c5}
       \field{extraname}{1}
       \field{sortinit}{1}
-      \field{sortinithash}{4f6aaa89bab872aa0999fec09ff8e98a}
+      \field{sortinithash}{50c6687d7fc80f50136d75228e3c59ba}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -95,7 +95,7 @@ my $k2 = q|    \entry{key2}{inbook}{}
       \strng{authorfullhash}{a517747c3d12f99244ae598910d979c5}
       \field{extraname}{2}
       \field{sortinit}{2}
-      \field{sortinithash}{8b555b3791beccb63322c22f3320aa9a}
+      \field{sortinithash}{ed39bb39cf854d5250e95b1c1f94f4ed}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -254,7 +254,7 @@ my $kck4 = q|    \entry{caf8e34be07426ae7127c1b4829983c1}{inbook}{skipbib=true,s
 
 my $c1 = q|    \entry{c1}{book}{}
       \field{sortinit}{3}
-      \field{sortinithash}{ad6fe7482ffbd7b9f99c9e8b5dccd3d7}
+      \field{sortinithash}{a37a8ef248a93c322189792c34fc68c9}
       \field{related}{9ab62b5ef34a985438bfdf7ee0102229}
     \endentry
 |;
@@ -341,7 +341,7 @@ my $un1 = q|    \entry{kullback}{book}{}
       \strng{authorfullhash}{34c5bbf9876c37127c3abe4e7d7a7198}
       \field{extraname}{1}
       \field{sortinit}{5}
-      \field{sortinithash}{20e9b4b0b173788c5dace24730f47d8c}
+      \field{sortinithash}{5dd416adbafacc8226114bc0202d5fdd}
       \field{extradatescope}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -375,7 +375,7 @@ my $un2 = q|    \entry{kullback:related}{book}{}
       \strng{authorfullhash}{34c5bbf9876c37127c3abe4e7d7a7198}
       \field{extraname}{2}
       \field{sortinit}{6}
-      \field{sortinithash}{b33bc299efb3c36abec520a4c896a66d}
+      \field{sortinithash}{7851c86048328b027313775d8fbd2131}
       \field{extradatescope}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
diff --git t/remote-files.t t/remote-files.t
index ee83373a..3adaa239 100644
--- t/remote-files.t
+++ t/remote-files.t
@@ -74,7 +74,7 @@ my $dl1 = q|    \entry{AbdelbarH98}{article}{}
       \strng{authornamehash}{bb887c5d0458bfb1f3f7e6afc8d1def4}
       \strng{authorfullhash}{bb887c5d0458bfb1f3f7e6afc8d1def4}
       \field{sortinit}{A}
-      \field{sortinithash}{2f401846e2029bad6b3ecc16d50031e2}
+      \field{sortinithash}{a3dcedd53b04d1adfd5ac303ecd5e6fa}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
diff --git t/set-dynamic.t t/set-dynamic.t
index e837b97c..12f06ecb 100644
--- t/set-dynamic.t
+++ t/set-dynamic.t
@@ -51,7 +51,7 @@ my $out = $biber->get_output_obj;
 my $string1 = q|    \entry{DynSet}{set}{}
       \set{Dynamic1,Dynamic2,Dynamic3}
       \field{sortinit}{1}
-      \field{sortinithash}{4f6aaa89bab872aa0999fec09ff8e98a}
+      \field{sortinithash}{50c6687d7fc80f50136d75228e3c59ba}
     \endentry
 |;
 
@@ -71,7 +71,7 @@ my $string2 = q|    \entry{Dynamic1}{book}{skipbib=true,skipbiblist=true,skiplab
       \strng{authornamehash}{252caa7921a061ca92087a1a52f15b78}
       \strng{authorfullhash}{252caa7921a061ca92087a1a52f15b78}
       \field{sortinit}{8}
-      \field{sortinithash}{a231b008ebf0ecbe0b4d96dcc159445f}
+      \field{sortinithash}{1b24cab5087933ef0826a7cd3b99e994}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -98,7 +98,7 @@ my $string3 = q|    \entry{Dynamic2}{book}{skipbib=true,skipbiblist=true,skiplab
       \strng{authornamehash}{894a5fe6de820f5dcce84a65581667f4}
       \strng{authorfullhash}{894a5fe6de820f5dcce84a65581667f4}
       \field{sortinit}{9}
-      \field{sortinithash}{0a5ebc79d83c96b6579069544c73c7d4}
+      \field{sortinithash}{54047ffb55bdefa0694bbd554c1b11a0}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -124,7 +124,7 @@ my $string4 = q|    \entry{Dynamic3}{book}{skipbib=true,skipbiblist=true,skiplab
       \strng{authornamehash}{fc3cc97631ceaecdde2aee6cc60ab42b}
       \strng{authorfullhash}{fc3cc97631ceaecdde2aee6cc60ab42b}
       \field{sortinit}{1}
-      \field{sortinithash}{4f6aaa89bab872aa0999fec09ff8e98a}
+      \field{sortinithash}{50c6687d7fc80f50136d75228e3c59ba}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -151,7 +151,7 @@ my $string5 = q|    \entry{Dynamic3}{book}{}
       \strng{authornamehash}{fc3cc97631ceaecdde2aee6cc60ab42b}
       \strng{authorfullhash}{fc3cc97631ceaecdde2aee6cc60ab42b}
       \field{sortinit}{1}
-      \field{sortinithash}{4f6aaa89bab872aa0999fec09ff8e98a}
+      \field{sortinithash}{50c6687d7fc80f50136d75228e3c59ba}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
diff --git t/set-legacy.t t/set-legacy.t
index a1346d33..28b1563d 100644
--- t/set-legacy.t
+++ t/set-legacy.t
@@ -45,7 +45,7 @@ my $out = $biber->get_output_obj;
 my $string1 = q|    \entry{Elias1955}{set}{}
       \set{Elias1955a,Elias1955b}
       \field{sortinit}{1}
-      \field{sortinithash}{4f6aaa89bab872aa0999fec09ff8e98a}
+      \field{sortinithash}{50c6687d7fc80f50136d75228e3c59ba}
     \endentry
 |;
 
diff --git t/set-static.t t/set-static.t
index 27843949..87cd66f3 100644
--- t/set-static.t
+++ t/set-static.t
@@ -47,7 +47,7 @@ my $out = $biber->get_output_obj;
 my $string1 = q|    \entry{Static1}{set}{}
       \set{Static2,Static4,Static3}
       \field{sortinit}{B}
-      \field{sortinithash}{d7095fff47cda75ca2589920aae98399}
+      \field{sortinithash}{8de16967003c7207dae369d874f1456e}
       \field{annotation}{Some notes}
     \endentry
 |;
@@ -68,7 +68,7 @@ my $string2 = q|    \entry{Static2}{book}{skipbib=true,skipbiblist=true,skiplab=
       \strng{authornamehash}{43874d80d7ce68027102819f16c47df1}
       \strng{authorfullhash}{43874d80d7ce68027102819f16c47df1}
       \field{sortinit}{B}
-      \field{sortinithash}{d7095fff47cda75ca2589920aae98399}
+      \field{sortinithash}{8de16967003c7207dae369d874f1456e}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -94,7 +94,7 @@ my $string3 = q|    \entry{Static3}{book}{skipbib=true,skipbiblist=true,skiplab=
       \strng{authornamehash}{22dafa5cd57bb5dd7f3e3bab98fd539c}
       \strng{authorfullhash}{22dafa5cd57bb5dd7f3e3bab98fd539c}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -119,7 +119,7 @@ my $string4 = q|    \entry{Static4}{book}{skipbib=true,skipbiblist=true,skiplab=
       \strng{authornamehash}{da80091c8cd89e5269bd55af1bd5d2fa}
       \strng{authorfullhash}{da80091c8cd89e5269bd55af1bd5d2fa}
       \field{sortinit}{C}
-      \field{sortinithash}{4d103a86280481745c9c897c925753c0}
+      \field{sortinithash}{4c244ceae61406cdc0cc2ce1cb1ff703}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -145,7 +145,7 @@ my $string5 = q|    \entry{Static2}{book}{}
       \strng{authornamehash}{43874d80d7ce68027102819f16c47df1}
       \strng{authorfullhash}{43874d80d7ce68027102819f16c47df1}
       \field{sortinit}{1}
-      \field{sortinithash}{4f6aaa89bab872aa0999fec09ff8e98a}
+      \field{sortinithash}{50c6687d7fc80f50136d75228e3c59ba}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
diff --git t/skips.t t/skips.t
index 3c01d9e2..eb646c6e 100644
--- t/skips.t
+++ t/skips.t
@@ -50,7 +50,7 @@ my $set1 = q|    \entry{seta}{set}{}
       \field{labelalpha}{Doe10}
       \field{extraalpha}{1}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
     \endentry
 |;
 
@@ -70,7 +70,7 @@ my $set2 = q|    \entry{set:membera}{book}{skipbib=true,skipbiblist=true,skiplab
       \strng{authornamehash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -97,7 +97,7 @@ my $set3 = q|    \entry{set:memberb}{book}{skipbib=true,skipbiblist=true,skiplab
       \strng{authornamehash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -123,7 +123,7 @@ my $set4 = q|    \entry{set:memberc}{book}{skipbib=true,skipbiblist=true,skiplab
       \strng{authornamehash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -150,7 +150,7 @@ my $noset1 = q|    \entry{noseta}{book}{}
       \field{extraname}{3}
       \field{labelalpha}{Doe10}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extradate}{2}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
@@ -180,7 +180,7 @@ my $noset2 = q|    \entry{nosetb}{book}{}
       \field{extraname}{4}
       \field{labelalpha}{Doe10}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extradate}{3}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
@@ -210,7 +210,7 @@ my $noset3 = q|    \entry{nosetc}{book}{}
       \field{extraname}{5}
       \field{labelalpha}{Doe10}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extradate}{4}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
@@ -244,7 +244,7 @@ my $sk4 = q|    \entry{skip4}{article}{skipbib=true,skipbiblist=true,skiplab=tru
       \strng{authornamehash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
diff --git t/skipsg.t t/skipsg.t
index f7909281..bec01ab8 100644
--- t/skipsg.t
+++ t/skipsg.t
@@ -65,7 +65,7 @@ my $S1 = q|    \entry{S1}{book}{skipbib=false,skipbiblist=false,skiplab=false}
       \field{extraname}{1}
       \field{labelalpha}{DA95}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extradate}{1}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{year}
@@ -102,7 +102,7 @@ my $S2 = q|    \entry{S2}{book}{skipbib=false,skiplab=false}
       \field{extraname}{2}
       \field{labelalpha}{DA95}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extradate}{2}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{year}
@@ -137,7 +137,7 @@ my $S3 = q|    \entry{S3}{book}{}
       \strng{authornamehash}{8c77336299b25bdada7bf8038f46722f}
       \strng{authorfullhash}{8c77336299b25bdada7bf8038f46722f}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{labeldatesource}{year}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
diff --git t/sort-complex.t t/sort-complex.t
index 72ffb965..1913041c 100644
--- t/sort-complex.t
+++ t/sort-complex.t
@@ -123,7 +123,7 @@ my $l4 = q|    \entry{L4}{book}{}
       \field{extraname}{2}
       \field{labelalpha}{Doe\textbf{+}95}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extraalpha}{2}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -155,7 +155,7 @@ my $l1 = q|    \entry{L1}{book}{}
       \field{extraname}{1}
       \field{labelalpha}{Doe95}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extraalpha}{1}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -187,7 +187,7 @@ my $l2 = q|    \entry{L2}{book}{}
       \field{extraname}{3}
       \field{labelalpha}{Doe95}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extraalpha}{3}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -219,7 +219,7 @@ my $l3 = q|    \entry{L3}{book}{}
       \field{extraname}{2}
       \field{labelalpha}{Doe95}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extraalpha}{2}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -253,7 +253,7 @@ my $l5 = q|    \entry{L5}{book}{}
       \field{extraname}{1}
       \field{labelalpha}{Doe\textbf{+}95}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \field{extraalpha}{1}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
diff --git t/tdata/full-bbl.bbl t/tdata/full-bbl.bbl
index c263c70b..5e63198f 100644
--- t/tdata/full-bbl.bbl
+++ t/tdata/full-bbl.bbl
@@ -36,7 +36,7 @@
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{labelalpha}{\emph{A}}
       \field{sortinit}{A}
-      \field{sortinithash}{2f401846e2029bad6b3ecc16d50031e2}
+      \field{sortinithash}{a3dcedd53b04d1adfd5ac303ecd5e6fa}
       \true{singletitle}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -63,7 +63,7 @@
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{labelalpha}{\emph{A}}
       \field{sortinit}{A}
-      \field{sortinithash}{2f401846e2029bad6b3ecc16d50031e2}
+      \field{sortinithash}{a3dcedd53b04d1adfd5ac303ecd5e6fa}
       \true{singletitle}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
@@ -90,7 +90,7 @@
       \strng{authorfullhash}{bd051a2f7a5f377e3a62581b0e0f8577}
       \field{labelalpha}{\emph{A}}
       \field{sortinit}{D}
-      \field{sortinithash}{6f385f66841fb5e82009dc833c761848}
+      \field{sortinithash}{c438b3d5d027251ba63f5ed538d98af5}
       \true{singletitle}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
diff --git t/tdata/full-bblxml.bblxml t/tdata/full-bblxml.bblxml
index 8f6de672..42fbfa3f 100644
--- t/tdata/full-bblxml.bblxml
+++ t/tdata/full-bblxml.bblxml
@@ -47,7 +47,7 @@
         <bbl:field name="authorbibnamehash">860705eb8ffac6e40507746c7ede79ca</bbl:field>
         <bbl:field name="labelalpha">Aks\textbf{+}06</bbl:field>
         <bbl:field name="sortinit">A</bbl:field>
-        <bbl:field name="sortinithash">2f401846e2029bad6b3ecc16d50031e2</bbl:field>
+        <bbl:field name="sortinithash">a3dcedd53b04d1adfd5ac303ecd5e6fa</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="indextitle">Effect of immobilization on catalytic characteristics</bbl:field>
@@ -79,7 +79,7 @@
         <bbl:field name="authorbibnamehash">9717d2d959a4b18727868ca7928c34b4</bbl:field>
         <bbl:field name="labelalpha">Ang02</bbl:field>
         <bbl:field name="sortinit">A</bbl:field>
-        <bbl:field name="sortinithash">2f401846e2029bad6b3ecc16d50031e2</bbl:field>
+        <bbl:field name="sortinithash">a3dcedd53b04d1adfd5ac303ecd5e6fa</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">shorttitle</bbl:field>
         <bbl:field name="annotation">A German article in a French journal. Apart from that, a typical \texttt{article} entry. Note the \texttt{indextitle} field</bbl:field>
@@ -109,7 +109,7 @@
         </bbl:set>
         <bbl:field name="labelalpha">Gla61</bbl:field>
         <bbl:field name="sortinit">G</bbl:field>
-        <bbl:field name="sortinithash">32d67eca0634bf53703493fb1090a2e8</bbl:field>
+        <bbl:field name="sortinithash">62eb2aa29549e4fdbd3cb154ec5711cb</bbl:field>
         <bbl:field name="annotation">A set entry with three members discussing the standard model of particle physics.</bbl:field>
       </bbl:entry>
       <bbl:entry key="stdmodel:glashow" type="article">
@@ -136,7 +136,7 @@
         <bbl:field name="authorfullhash">eb3f242602109f5bdbeb41fdd990fcc0</bbl:field>
         <bbl:field name="authorbibnamehash">eb3f242602109f5bdbeb41fdd990fcc0</bbl:field>
         <bbl:field name="sortinit">G</bbl:field>
-        <bbl:field name="sortinithash">32d67eca0634bf53703493fb1090a2e8</bbl:field>
+        <bbl:field name="sortinithash">62eb2aa29549e4fdbd3cb154ec5711cb</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="journaltitle">Nucl.~Phys.</bbl:field>
@@ -172,7 +172,7 @@
         <bbl:field name="extraname">1</bbl:field>
         <bbl:field name="labelalpha">Knu86</bbl:field>
         <bbl:field name="sortinit">K</bbl:field>
-        <bbl:field name="sortinithash">c02bf6bff1c488450c352b40f5d853ab</bbl:field>
+        <bbl:field name="sortinithash">d3edc18d54b9438a72c24c925bfb38f4</bbl:field>
         <bbl:field name="extraalpha">1</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">shorttitle</bbl:field>
@@ -208,7 +208,7 @@
         <bbl:field name="extraname">2</bbl:field>
         <bbl:field name="labelalpha">Knu86</bbl:field>
         <bbl:field name="sortinit">K</bbl:field>
-        <bbl:field name="sortinithash">c02bf6bff1c488450c352b40f5d853ab</bbl:field>
+        <bbl:field name="sortinithash">d3edc18d54b9438a72c24c925bfb38f4</bbl:field>
         <bbl:field name="extraalpha">2</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">shorttitle</bbl:field>
@@ -237,7 +237,7 @@
         <bbl:field name="authorbibnamehash">d52f05e0199eae563725b4165f4b2520</bbl:field>
         <bbl:field name="labelalpha">Ree58</bbl:field>
         <bbl:field name="sortinit">R</bbl:field>
-        <bbl:field name="sortinithash">5e1c39a9d46ffb6bebd8f801023a9486</bbl:field>
+        <bbl:field name="sortinithash">b9c68a358aea118dfa887b6e902414a7</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="annotation">An \texttt{article} entry with a \texttt{series} and a \texttt{volume} field. Note the format of the series. If the value of the \texttt{series} field is an integer, this number is printed as an ordinal and the string \enquote*{series} is appended automatically</bbl:field>
@@ -294,7 +294,7 @@
         <bbl:field name="editorfullhash">d0ec890e59163c24c111a08d2a4be982</bbl:field>
         <bbl:field name="editorbibnamehash">d0ec890e59163c24c111a08d2a4be982</bbl:field>
         <bbl:field name="sortinit">S</bbl:field>
-        <bbl:field name="sortinithash">b164b07b29984b41daf1e85279fbc5ab</bbl:field>
+        <bbl:field name="sortinithash">c319cff79d99c853d775f88277d4e45f</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="booksubtitle">Relativistic groups and analyticity</bbl:field>
@@ -325,7 +325,7 @@
         <bbl:field name="authorbibnamehash">93f17724cbd884231ef39a73755ef5bb</bbl:field>
         <bbl:field name="labelalpha">Sho91</bbl:field>
         <bbl:field name="sortinit">S</bbl:field>
-        <bbl:field name="sortinithash">b164b07b29984b41daf1e85279fbc5ab</bbl:field>
+        <bbl:field name="sortinithash">c319cff79d99c853d775f88277d4e45f</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="annotation">An \texttt{article} entry with \texttt{series}, \texttt{volume}, and \texttt{number} fields. Note the format of the \texttt{series} which is a localization key</bbl:field>
@@ -368,7 +368,7 @@
         <bbl:field name="authorfullhash">15e144c72b0b9d10db00ddc6d37a19e6</bbl:field>
         <bbl:field name="authorbibnamehash">15e144c72b0b9d10db00ddc6d37a19e6</bbl:field>
         <bbl:field name="sortinit">W</bbl:field>
-        <bbl:field name="sortinithash">4315d78024d0cea9b57a0c6f0e35ed0d</bbl:field>
+        <bbl:field name="sortinithash">1af34bd8c148ffb32de1494636b49713</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="journaltitle">Phys.~Rev.~Lett.</bbl:field>
@@ -426,7 +426,7 @@
         <bbl:field name="authorbibnamehash">860705eb8ffac6e40507746c7ede79ca</bbl:field>
         <bbl:field name="labelalpha">Aks\textbf{+}06</bbl:field>
         <bbl:field name="sortinit">A</bbl:field>
-        <bbl:field name="sortinithash">2f401846e2029bad6b3ecc16d50031e2</bbl:field>
+        <bbl:field name="sortinithash">a3dcedd53b04d1adfd5ac303ecd5e6fa</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="indextitle">Effect of immobilization on catalytic characteristics</bbl:field>
@@ -458,7 +458,7 @@
         <bbl:field name="authorbibnamehash">9717d2d959a4b18727868ca7928c34b4</bbl:field>
         <bbl:field name="labelalpha">Ang02</bbl:field>
         <bbl:field name="sortinit">A</bbl:field>
-        <bbl:field name="sortinithash">2f401846e2029bad6b3ecc16d50031e2</bbl:field>
+        <bbl:field name="sortinithash">a3dcedd53b04d1adfd5ac303ecd5e6fa</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">shorttitle</bbl:field>
         <bbl:field name="annotation">A German article in a French journal. Apart from that, a typical \texttt{article} entry. Note the \texttt{indextitle} field</bbl:field>
@@ -488,7 +488,7 @@
         </bbl:set>
         <bbl:field name="labelalpha">Gla61</bbl:field>
         <bbl:field name="sortinit">G</bbl:field>
-        <bbl:field name="sortinithash">32d67eca0634bf53703493fb1090a2e8</bbl:field>
+        <bbl:field name="sortinithash">62eb2aa29549e4fdbd3cb154ec5711cb</bbl:field>
         <bbl:field name="annotation">A set entry with three members discussing the standard model of particle physics.</bbl:field>
       </bbl:entry>
       <bbl:entry key="stdmodel:glashow" type="article">
@@ -515,7 +515,7 @@
         <bbl:field name="authorfullhash">eb3f242602109f5bdbeb41fdd990fcc0</bbl:field>
         <bbl:field name="authorbibnamehash">eb3f242602109f5bdbeb41fdd990fcc0</bbl:field>
         <bbl:field name="sortinit">G</bbl:field>
-        <bbl:field name="sortinithash">32d67eca0634bf53703493fb1090a2e8</bbl:field>
+        <bbl:field name="sortinithash">62eb2aa29549e4fdbd3cb154ec5711cb</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="journaltitle">Nucl.~Phys.</bbl:field>
@@ -551,7 +551,7 @@
         <bbl:field name="extraname">1</bbl:field>
         <bbl:field name="labelalpha">Knu86</bbl:field>
         <bbl:field name="sortinit">K</bbl:field>
-        <bbl:field name="sortinithash">c02bf6bff1c488450c352b40f5d853ab</bbl:field>
+        <bbl:field name="sortinithash">d3edc18d54b9438a72c24c925bfb38f4</bbl:field>
         <bbl:field name="extraalpha">1</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">shorttitle</bbl:field>
@@ -587,7 +587,7 @@
         <bbl:field name="extraname">2</bbl:field>
         <bbl:field name="labelalpha">Knu86</bbl:field>
         <bbl:field name="sortinit">K</bbl:field>
-        <bbl:field name="sortinithash">c02bf6bff1c488450c352b40f5d853ab</bbl:field>
+        <bbl:field name="sortinithash">d3edc18d54b9438a72c24c925bfb38f4</bbl:field>
         <bbl:field name="extraalpha">2</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">shorttitle</bbl:field>
@@ -616,7 +616,7 @@
         <bbl:field name="authorbibnamehash">d52f05e0199eae563725b4165f4b2520</bbl:field>
         <bbl:field name="labelalpha">Ree58</bbl:field>
         <bbl:field name="sortinit">R</bbl:field>
-        <bbl:field name="sortinithash">5e1c39a9d46ffb6bebd8f801023a9486</bbl:field>
+        <bbl:field name="sortinithash">b9c68a358aea118dfa887b6e902414a7</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="annotation">An \texttt{article} entry with a \texttt{series} and a \texttt{volume} field. Note the format of the series. If the value of the \texttt{series} field is an integer, this number is printed as an ordinal and the string \enquote*{series} is appended automatically</bbl:field>
@@ -673,7 +673,7 @@
         <bbl:field name="editorfullhash">d0ec890e59163c24c111a08d2a4be982</bbl:field>
         <bbl:field name="editorbibnamehash">d0ec890e59163c24c111a08d2a4be982</bbl:field>
         <bbl:field name="sortinit">S</bbl:field>
-        <bbl:field name="sortinithash">b164b07b29984b41daf1e85279fbc5ab</bbl:field>
+        <bbl:field name="sortinithash">c319cff79d99c853d775f88277d4e45f</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="booksubtitle">Relativistic groups and analyticity</bbl:field>
@@ -704,7 +704,7 @@
         <bbl:field name="authorbibnamehash">93f17724cbd884231ef39a73755ef5bb</bbl:field>
         <bbl:field name="labelalpha">Sho91</bbl:field>
         <bbl:field name="sortinit">S</bbl:field>
-        <bbl:field name="sortinithash">b164b07b29984b41daf1e85279fbc5ab</bbl:field>
+        <bbl:field name="sortinithash">c319cff79d99c853d775f88277d4e45f</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="annotation">An \texttt{article} entry with \texttt{series}, \texttt{volume}, and \texttt{number} fields. Note the format of the \texttt{series} which is a localization key</bbl:field>
@@ -747,7 +747,7 @@
         <bbl:field name="authorfullhash">15e144c72b0b9d10db00ddc6d37a19e6</bbl:field>
         <bbl:field name="authorbibnamehash">15e144c72b0b9d10db00ddc6d37a19e6</bbl:field>
         <bbl:field name="sortinit">W</bbl:field>
-        <bbl:field name="sortinithash">4315d78024d0cea9b57a0c6f0e35ed0d</bbl:field>
+        <bbl:field name="sortinithash">1af34bd8c148ffb32de1494636b49713</bbl:field>
         <bbl:field name="labelnamesource">author</bbl:field>
         <bbl:field name="labeltitlesource">title</bbl:field>
         <bbl:field name="journaltitle">Phys.~Rev.~Lett.</bbl:field>
diff --git t/truncation.t t/truncation.t
index 74550728..575c64fc 100644
--- t/truncation.t
+++ t/truncation.t
@@ -63,7 +63,7 @@ my $us1 = q|    \entry{us1}{book}{}
       \strng{authorfullhash}{6a9b0705c275273262103333472cc656}
       \field{labelalpha}{Elk72}
       \field{sortinit}{E}
-      \field{sortinithash}{8da8a182d344d5b9047633dfc0cc9131}
+      \field{sortinithash}{c554bd1a0b76ea92b9f105fe36d9c7b0}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -93,7 +93,7 @@ my $us2a = q|    \entry{us2}{book}{}
       \strng{authorfullhash}{40a337fc8d6319ae5a7b50f6324781ec}
       \field{labelalpha}{Elk\textbf{+}72}
       \field{sortinit}{E}
-      \field{sortinithash}{8da8a182d344d5b9047633dfc0cc9131}
+      \field{sortinithash}{c554bd1a0b76ea92b9f105fe36d9c7b0}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -121,7 +121,7 @@ my $us3 = q|    \entry{us3}{book}{}
       \strng{authorfullhash}{e06f6e5a8c1d5204dea326aa5f4f8d17}
       \field{labelalpha}{Uth00}
       \field{sortinit}{U}
-      \field{sortinithash}{6901a00e45705986ee5e7ca9fd39adca}
+      \field{sortinithash}{77a6935510e008adcf5b555e7b4f0711}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -169,7 +169,7 @@ my $us4a = q|    \entry{us4}{book}{}
       \field{extraname}{1}
       \field{labelalpha}{Uth\textbf{+}00}
       \field{sortinit}{U}
-      \field{sortinithash}{6901a00e45705986ee5e7ca9fd39adca}
+      \field{sortinithash}{77a6935510e008adcf5b555e7b4f0711}
       \field{extradate}{1}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
@@ -202,7 +202,7 @@ my $us2b = q|    \entry{us2}{book}{}
       \field{extraname}{2}
       \field{labelalpha}{Elk\textbf{+}72}
       \field{sortinit}{E}
-      \field{sortinithash}{8da8a182d344d5b9047633dfc0cc9131}
+      \field{sortinithash}{c554bd1a0b76ea92b9f105fe36d9c7b0}
       \field{extradate}{2}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
@@ -251,7 +251,7 @@ my $us4b = q|    \entry{us4}{book}{}
       \field{extraname}{2}
       \field{labelalpha}{Uth\textbf{+}00}
       \field{sortinit}{U}
-      \field{sortinithash}{6901a00e45705986ee5e7ca9fd39adca}
+      \field{sortinithash}{77a6935510e008adcf5b555e7b4f0711}
       \field{extradate}{2}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
@@ -281,7 +281,7 @@ my $us6 = q|    \entry{us6}{book}{}
       \strng{authorfullhash}{cbe9a5912d961199801c3fcd32356ecf}
       \field{labelalpha}{Red71}
       \field{sortinit}{R}
-      \field{sortinithash}{5e1c39a9d46ffb6bebd8f801023a9486}
+      \field{sortinithash}{b9c68a358aea118dfa887b6e902414a7}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -311,7 +311,7 @@ my $us7 = q|    \entry{us7}{misc}{}
       \strng{authorfullhash}{d70785a70cdf36c7b5dc7b136207ada9}
       \field{labelalpha}{Red\textbf{+}71}
       \field{sortinit}{R}
-      \field{sortinithash}{5e1c39a9d46ffb6bebd8f801023a9486}
+      \field{sortinithash}{b9c68a358aea118dfa887b6e902414a7}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -340,7 +340,7 @@ my $us8 = q|    \entry{us8}{book}{}
       \field{extraname}{1}
       \field{labelalpha}{Sly00}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradate}{1}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
@@ -387,7 +387,7 @@ my $us9 = q|    \entry{us9}{book}{}
       \strng{authorfullhash}{afe15ce8d7d22d0bbc042705c4b5fdf6}
       \field{labelalpha}{Sly\textbf{+}00}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{extraalpha}{1}
diff --git t/uniqueness-nameparts.t t/uniqueness-nameparts.t
index 44c4b5e1..9af0fb06 100644
--- t/uniqueness-nameparts.t
+++ t/uniqueness-nameparts.t
@@ -81,7 +81,7 @@ my $un1 = q|    \entry{un1}{article}{}
       \strng{authorfullhash}{329d8f9192ea3349d700160c9ddb505d}
       \field{labelalpha}{SmiJohSim}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -106,7 +106,7 @@ my $un2 = q|    \entry{un2}{article}{}
       \strng{authorfullhash}{7551114aede4ef69e4b3683039801706}
       \field{labelalpha}{SmiJohAla}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -131,7 +131,7 @@ my $un3 = q|    \entry{un3}{article}{}
       \strng{authorfullhash}{401aebda288799a7c757526242d8c9fc}
       \field{labelalpha}{SmiJohArt}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -156,7 +156,7 @@ my $un4 = q|    \entry{un4}{article}{}
       \strng{authorfullhash}{f6038a264619efefd49c7daac56424ca}
       \field{labelalpha}{SmiAlaSim}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -181,7 +181,7 @@ my $un1a = q|    \entry{un1}{article}{}
       \strng{authorfullhash}{329d8f9192ea3349d700160c9ddb505d}
       \field{labelalpha}{SmiJohSim}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -206,7 +206,7 @@ my $un2a = q|    \entry{un2}{article}{}
       \strng{authorfullhash}{7551114aede4ef69e4b3683039801706}
       \field{labelalpha}{SmiJohAla}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -231,7 +231,7 @@ my $un3a = q|    \entry{un3}{article}{}
       \strng{authorfullhash}{401aebda288799a7c757526242d8c9fc}
       \field{labelalpha}{SmiJohArt}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -257,7 +257,7 @@ my $un4a = q|    \entry{un4}{article}{}
       \strng{authorfullhash}{f6038a264619efefd49c7daac56424ca}
       \field{labelalpha}{SmiAlaSim}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -285,7 +285,7 @@ my $un1b = q|    \entry{un1}{article}{}
       \field{extraname}{5}
       \field{labelalpha}{SmiJohSim}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradate}{5}
       \field{labelnamesource}{author}
     \endentry
@@ -311,7 +311,7 @@ my $un2b = q|    \entry{un2}{article}{}
       \strng{authorfullhash}{7551114aede4ef69e4b3683039801706}
       \field{labelalpha}{SmiJohAla}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -336,7 +336,7 @@ my $un3b = q|    \entry{un3}{article}{}
       \strng{authorfullhash}{401aebda288799a7c757526242d8c9fc}
       \field{labelalpha}{SmiJohArt}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{labelnamesource}{author}
     \endentry
 |;
@@ -364,7 +364,7 @@ my $un4b = q|    \entry{un4}{article}{}
       \field{extraname}{1}
       \field{labelalpha}{SmiAlaSim}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradate}{1}
       \field{labelnamesource}{author}
     \endentry
@@ -393,7 +393,7 @@ my $un5 = q|    \entry{un5}{article}{uniquenametemplatename=test3}
       \field{extraname}{2}
       \field{labelalpha}{SmiArtSim}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradate}{2}
       \field{labelnamesource}{author}
     \endentry
@@ -422,7 +422,7 @@ my $un6 = q|    \entry{un6}{article}{}
       \field{extraname}{3}
       \field{labelalpha}{SmiArtSmy}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradate}{3}
       \field{labelnamesource}{author}
     \\endentry
@@ -451,7 +451,7 @@ my $un7 = q|    \entry{un7}{article}{}
       \field{extraname}{4}
       \field{labelalpha}{SmiArtSme}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{extradate}{4}
       \field{labelnamesource}{author}
     \endentry
diff --git t/xdata.t t/xdata.t
index ece3f1e5..3c683a8b 100644
--- t/xdata.t
+++ t/xdata.t
@@ -79,7 +79,7 @@ my $xd1 = q|    \entry{xd1}{book}{}
       \strng{authorfullhash}{51db4bfd331cba22959ce2d224c517cd}
       \field{extraname}{2}
       \field{sortinit}{E}
-      \field{sortinithash}{8da8a182d344d5b9047633dfc0cc9131}
+      \field{sortinithash}{c554bd1a0b76ea92b9f105fe36d9c7b0}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -112,7 +112,7 @@ my $xd2 = q|    \entry{xd2}{book}{}
       \strng{authornamehash}{68539e0ce4922cc4957c6cabf35e6fc8}
       \strng{authorfullhash}{68539e0ce4922cc4957c6cabf35e6fc8}
       \field{sortinit}{P}
-      \field{sortinithash}{ff3bcf24f47321b42cb156c2cc8a8422}
+      \field{sortinithash}{bb5b15f2db90f7aef79bb9e83defefcb}
       \field{extradatescope}{labelyear}
       \field{labeldatesource}{}
       \field{labelnamesource}{author}
@@ -182,7 +182,7 @@ my $gxd1 = q|    \entry{gxd1}{book}{}
       \strng{translatornamehash}{d41d8cd98f00b204e9800998ecf8427e}
       \strng{translatorfullhash}{d41d8cd98f00b204e9800998ecf8427e}
       \field{sortinit}{S}
-      \field{sortinithash}{b164b07b29984b41daf1e85279fbc5ab}
+      \field{sortinithash}{c319cff79d99c853d775f88277d4e45f}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
       \field{addendum}{xdata=missing}
@@ -248,7 +248,7 @@ my $bltxgxd1 = q|    \entry{bltxgxd1}{book}{}
       \strng{translatornamehash}{d41d8cd98f00b204e9800998ecf8427e}
       \strng{translatorfullhash}{d41d8cd98f00b204e9800998ecf8427e}
       \field{sortinit}{B}
-      \field{sortinithash}{d7095fff47cda75ca2589920aae98399}
+      \field{sortinithash}{8de16967003c7207dae369d874f1456e}
       \field{labelnamesource}{author}
       \field{labeltitlesource}{title}
       \field{addendum}{xdata=missing}
